Source/WebCore/page/DOMWindow.idl

```diff
@@ -10,20 +11,42 @@
  *    notice, this list of conditions and the following disclaimer in the
  *    documentation and/or other materials provided with the distribution.
  *
- * THIS SOFTWARE IS PROVIDED BY APPLE INC. AND ITS CONTRIBUTORS ``AS IS''
```

Comment:
This seems like it shouldn't be changed. I think the old license was correct.

Reply:
I'm moving LocalDOMWindow.idl to DOMWindow.idl and retaining the exact license of LocalDOMWindow.idl